### DISCUSSION TITLE
How do I keep an empty list in YAML output?

### DISCUSSION THREAD

**Discussion by nkabir** [kcl-lang/kcl#615] [General]
I'd like to output an empty list as

```yaml
mykey: []
```

Is it possible to prevent kcl from dropping this entry in output?

**Comment by Peefy** [comment]
Sorry, KCL's output of an empty list is controlled by a feature flag (`plan_empty_list`), which we will open it in KCL v0.5.1. Can you tell me which scenario the empty list output will be used for? Thank you for your discussion. @nkabir 
![image](https://github.com/kcl-lang/kcl/assets/22744597/a9c020a7-393b-4eec-9064-a0b729572387)

**Comment by nkabir** [comment]
Hi @Peefy 
Thank you for your excellent work!
When parsing the generated YAML, I wanted to avoid performing checks on the existence of keys.
